packages/webpack/src/runtime/runtime.js

```diff
@@ -48,30 +49,56 @@ const stricterScopeTerminator = freeze(
 /**
  * Enforces the policy for resource imports.
  *
- * @param {string | undefined} requestedResourceId - The ID of the requested
- *   resource.
+ * @template T
+ * @param {string} specifier - The ID of the requested resource.
  * @param {string} referrerResourceId - The ID of the referrer resource.
+ * @param {() => T} wrappedRequire - The wrapped **webpack_require** function.
+ * @returns {T} The result of the wrapped **webpack_require** function.
  * @throws {Error} Throws an error if the policy does not allow importing the
  *   requested resource from the referrer resource.
  */
-const enforcePolicy = (requestedResourceId, referrerResourceId) => {
-  if (typeof requestedResourceId === 'undefined') {
-    throw Error(`Requested resource ID is undefined`)
+const enforcePolicy = (specifier, referrerResourceId, wrappedRequire) => {
+  if (typeof specifier === 'undefined') {
+    throw Error(`Requested specifier is undefined`)
   }
-  requestedResourceId = '' + requestedResourceId
-  referrerResourceId = '' + referrerResourceId
-  // implicitly allow all for root and modules from the same package
+  // skip enforcing what we determined at build time we cannot
   if (
-    referrerResourceId === LAVAMOAT.root ||
-    requestedResourceId === referrerResourceId
+    LAVAMOAT.unenforceable.includes(specifier) ||
+    // implicitly allow all for root
+    referrerResourceId === LAVAMOAT.root
```

Comment:
is there a reason why `LAVAMOAT.unenforceable` is not a `Set`?  can this array get big enough to worry about?

Reply:
the content of this array should be limited to the names of node builtins that webpack5 won't polyfill. It can be a Set.